tdrs-backend/tdpservice/core/views.py

```diff
@@ -21,4 +26,15 @@ def write_logs(request):
                 f"{data['message']} "
                 f"for {request.user}")
 
+    if 'files' in data:
+        for file in data['files']:
+            LogEntry.objects.log_action(
+                user_id=User.objects.get(username=data['user']),
+                content_type_id=ContentType.objects.get_for_model(ReportFile).pk,
+                object_id=file,
+                object_repr=str(data),
+                action_flag=CHANGE,
+                change_message=data['message'],
```

Comment:
Is this going to make the change message `Submitted x data file(s)` for each one of these LogEntry records?

Reply:
Ahh yeah it will, good catch.

Reply:
I'll just make it `Submitted data file(s)`